protocol_driver_grpc.cc

```diff
@@ -295,8 +295,11 @@ absl::Status ProtocolDriverGrpc::Initialize(
     server_ =
         std::unique_ptr<ProtocolDriverServer>(new GrpcHandoffServerDriver());
   } else if (server_type == "polling") {
+    auto threadpool_size = GetNamedServerSettingInt64(
+        pd_opts, "threadpool_size", absl::base_internal::NumCPUs());
     server_ =
-        std::unique_ptr<ProtocolDriverServer>(new GrpcPollingServerDriver());
+        std::unique_ptr<ProtocolDriverServer>(
+            new GrpcPollingServerDriver(threadpool_size));
```

Comment:
good for now, but maybe we should have made this a struct of threadpool options.

Reply:
Added a TODO at GrpcPollingServerDriver's definition in protocol_driver_grpc.h